misk-jdbc/src/main/kotlin/misk/jdbc/DataSourceConfig.kt

```diff
@@ -76,7 +76,13 @@ data class DataSourceConfig(
   val show_sql: String? = "false",
   // Consider using this if you want Hibernate to automagically batch inserts/updates when it can.
   val jdbc_statement_batch_size: Int? = null,
-  val use_fixed_pool_size: Boolean = false
+  val use_fixed_pool_size: Boolean = false,
+  // MySQL 8+ by default requires authentication via RSA keys if TLS is unavailable.
+  // Within secured subnets, overriding to true can be acceptable.
+  // See https://mysqlconnector.net/troubleshooting/retrieval-public-key/
+  val allow_public_key_retrieval: Boolean = false,
+  // Allow setting additional JDBC url parameters for advanced configuration
+  val jdbc_url_query_parameters: Map<String, Any> = mapOf()
```

Comment:
yea I like this, makes it easy to change without having to send a misk PR, but you can always add in params after using them a while